### PR TITLE
feat: Create python image management

### PR DESCRIPTION
**Pull request by Mallear**
Create Python image management
---

- Generate Python image's Dockerfile based on Dockerfile template for PHP. 
- Install the bare minimum : basic system packages, CI Helper and Pipenv.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ matrix:
     - env: LANGUAGE=php           VERSION=5.3  PHP_VERSION=5.3.10 PHP_BUILD_INSTALL_EXTENSION=apcu=4.0.11 REDIS_VERSION=3.1.6 SECURITY_CHECKER_VERSION=2.0.6
     - env: LANGUAGE=php           VERSION=5.6  PHP_VERSION=5.6.35-cli-alpine APCU_VERSION=4.0.11 COMPOSER_VERSION=1.6.3 REDIS_VERSION=4.0.0 SECURITY_CHECKER_VERSION=4.1.6 XDEBUG_VERSION=XDEBUG_2_5_5
     - env: LANGUAGE=php           VERSION=7.1  PHP_VERSION=7.1.16-cli-alpine APCU_VERSION=5.1.11 COMPOSER_VERSION=1.6.3 REDIS_VERSION=4.0.0 SECURITY_CHECKER_VERSION=4.1.8 XDEBUG_VERSION=2.6.0
+    - env: LANGUAGE=python        VERSION=2.7.15  PYTHON_VERSION=2.7.15
```
Comment (mremi) [inline .travis.yml]: `VERSION` should be `2.7` (same for `3.6`)

File: travis.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ def main():
 
     run_build(buildInfo)
 
+
```
Comment (mremi) [inline travis.py]: Can you revert it please?

File: .travis.yml (hunk 1 above)
Comment (Mallear) [reply .travis.yml]: Done

File: travis.py (hunk 2 above)
Comment (Mallear) [reply travis.py]: I added it for best practices purpose: Python interpretor needs two blank lines before a new function declaration (same as the blank line at the end of the file).

**Pull request merged**